core/src/main/java/org/dashj/hash/X11.java

```diff
@@ -0,0 +1,104 @@
+package org.dashj.hash;
+
+import fr.cryptohash.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+
+/**
+ * Created by Hash Engineering on 4/24/14 for the X11 algorithm
+ */
+public class X11 {
+
+    private static final Logger log = LoggerFactory.getLogger(X11.class);
+    private static boolean native_library_loaded = false;
+
+    static BLAKE512 blake512 = new BLAKE512();
+    static BMW512 bmw512= new BMW512();
+    static Groestl512 groestl512 = new Groestl512();
+    static Skein512 skein512 = new Skein512();
+    static JH512 jh512 = new JH512();
+    static Keccak512 keccak512 = new Keccak512();
+    static Luffa512 luffa512 = new Luffa512();
+    static CubeHash512 cubehash512 = new CubeHash512();
+    static SHAvite512 shavite512 = new SHAvite512();
+    static SIMD512 simd512 = new SIMD512();
+    static ECHO512 echo512 = new ECHO512();
+
+    static {
+
+        try {
+            log.info("Loading x11 native library...");
+            System.loadLibrary("x11");
```

Comment:
If you have no need for Native Libraries, then we can leave this out.  The x11 library wouldn't necessarily be included in libdohj, but it would be included in the application that uses libdohj.  In my next revision of this pull request, I will remove this.